logging/logging-annotations/src/main/java/org/infinispan/logging/annotations/Description.java

```diff
@@ -0,0 +1,60 @@
+package org.infinispan.logging.annotations;
+
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.CLASS;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Descriptions should provide additional detail for each message and suggested user action.
+ * Generate HTML reference for log messages and descriptions. 
+ * Description.
```

Comment:
I think `Descriptions should provide additional detail for each message and suggested user action.` is sufficient. The annotation itself does not result in the HTML report being generated.